pageserver/src/tenant/timeline.rs

```diff
@@ -967,11 +969,26 @@ impl Timeline {
         lsn: Lsn,
         ctx: &RequestContext,
     ) -> Result<BTreeMap<Key, Result<Bytes, PageReconstructError>>, GetVectoredError> {
+        use crate::metrics::GetKind;
```

Comment:
This use should be at the top of the file.

Reply:
Sure: [af6627b](https://github.com/neondatabase/neon/pull/7486/commits/af6627b935920050dc938bbcbe0e80e32d29ac1b)